File: comparacion.c
#include <stdio.h>
int num1, num2, num3;
int main()
{
    printf("Ingresa el primer numero entero\n");
    scanf("%d", &num1);
    printf("Ingresa el segundo numero entero\n");
    scanf("%d", &num2);
    printf("Ingresa el tercer numero entero\n");
    scanf("%d", &num3);

    if (num1 > num2)
    printf("El numero %d es mayor a %d\n", num1, num2);
    else if (num1==num2){
        printf("El numero %d es igual a %d\n", num1, num2);
    }
    else{
        printf ("El numero %d es menor a %d\n", num1, num2);
    }
    if (num1 > num3)
    printf("El numero %d es mayor a %d\n", num1, num3);
    else if (num1==num3){
        printf("El numero %d es igual a %d\n", num1, num3);
    }
    else{
        printf("El numero %d es menor a %d\n", num1, num3);
    }
    if (num2 > num3)
    printf("El numero %d es mayor a %d\n", num2, num3);
    else if (num2==num3){
        printf("El numero %d es igual a %d\n", num2, num3);
    }
    else{
        printf("El numero %d es menor a %d\n", num2, num3);
    }
    return 0;
}

File: comparacion.py
num1 = float(input("Ingresa el primer numero\n"))
num2 = float(input("Ingresa el segundo numero\n"))
num3 = float(input("Ingresa el tercer numero\n"))

if num1 > num2:
    print(f"El numero {num1} es mayor que {num2}")
elif num1 < num2:
    print(f"El numero {num1} es menor que {num2}")
else:
    print(f"El numero {num1} es igual a {num2}")

if num1 > num3:
    print(f"El numero {num1} es mayor que {num3}")
elif num1 < num2:
    print(f"El numero {num1} es menor que {num3}")
else:
    print(f"El numero {num1} es igual a {num3}")

if num2 > num3:
    print(f"El numero {num2} es mayor que {num3}")
elif num1 < num2:
    print(f"El numero {num2} es menor que {num3}")
else:
    print(f"El numero {num2} es igual a {num3}")